src/Pos/Genesis/Types.hs

```diff
@@ -1,11 +1,14 @@
 module Pos.Genesis.Types
        ( StakeDistribution (..)
        , GenesisData (..)
+       , getTotalStake
        ) where
 
 import           Universum
 
-import           Pos.Core.Types                (Address, Coin)
+import           Pos.Core.Coin                 (coinToInteger, sumCoins, unsafeAddCoin,
```

Comment:
I think that these imports can be combined since `Pos.Core` reexports everything.